components/script/script_thread.rs

```diff
@@ -2102,6 +2102,7 @@ impl ScriptThread {
                 result,
             } => {
                 let global = self.documents.borrow().find_global(pipeline_id).unwrap();
+                let _ac = enter_realm(&*global);
```

Comment:
I'm not sure that `ac` is adding anything here...

```suggestion
                let _ = enter_realm(&*global);
```

Reply:
`let _` is not same as `let _x` see here: https://github.com/rust-lang/rust/issues/40096

`let _` does drop immediately (resolves must use warning), while `let _x` drops at the end. Also `let _ac` is also used in other servo code.